meta-xt-prod-aos-rcar-domd/recipes-aos/aos-provfinish/aos-provfinish.bbappend

```diff
@@ -0,0 +1,4 @@
+do_install_append() {
+    sed -i "s:/var/aos:/var:g" ${D}${aos_opt_dir}/provfinish.sh
```

Comment:
Here I'm not sure, but I suspect that this `provfinish.sh` was added by some 'not yet merged' commit previously. So maybe it's better to make this replacement in the original commit?
Maybe my assumption is wrong, but it seems that we are 'seding' our own file.

Reply:
It is part of meta-aos, an external meta-layer.